Author: Michael R. Crusoe <crusoe@debian.org>
Description: zlib-ng is not supported by Debian's flate2 package
Forwarded: not-needed
--- bgzip.orig/Cargo.toml
+++ bgzip/Cargo.toml
@@ -94,11 +94,3 @@
     "flate2/zlib",
     "flate2",
 ]
-zlib-ng = [
-    "flate2/zlib-ng",
-    "flate2",
-]
-zlib-ng-compat = [
-    "flate2/zlib-ng-compat",
-    "flate2",
-]
--- bgzip.orig/Cargo.toml.orig
+++ bgzip/Cargo.toml.orig
@@ -17,8 +17,6 @@
 flate2 = ["dep:flate2"]
 rust_backend = ["flate2/rust_backend", "flate2"]
 zlib = ["flate2/zlib", "flate2"]
-zlib-ng-compat = ["flate2/zlib-ng-compat", "flate2"]
-zlib-ng = ["flate2/zlib-ng", "flate2"]
 libdeflater = ["dep:libdeflater"]
 rayon = ["dep:rayon"]
 log  = ["dep:log"]
